src/LinuxTracing/SwitchesStatesNamesVisitor.h

```diff
@@ -59,6 +59,8 @@ class SwitchesStatesNamesVisitor : public PerfEventVisitor {
   void Visit(uint64_t event_timestamp, const TaskNewtaskPerfEventData& event_data) override;
   void Visit(uint64_t event_timestamp, const SchedSwitchPerfEventData& event_data) override;
   void Visit(uint64_t timestamp, const SchedWakeupPerfEventData& event_data) override;
+  void Visit(uint64_t timestamp, const SchedWakeupWithStackPerfEventData& event_data) override;
+  void Visit(uint64_t timestamp, const SchedSwitchWithStackPerfEventData& event_data) override;
```

Comment:
Weird order: either SchedSwitchPerfEventData, SchedWakeupPerfEventData, SchedSwitchWithStackPerfEventData, SchedWakeupWithStackPerfEventData; or SchedSwitchPerfEventData, SchedSwitchWithStackPerfEventData, SchedWakeupPerfEventData, SchedWakeupWithStackPerfEventData.

Reply:
Went with the second one.